dashboard/src/components/widget/footer/che-footer.directive.ts

```diff
@@ -87,7 +87,7 @@ export class CheFooter {
     }
 
     // docs
-    template += '<a class=\"che-footer-button-blue che-footer-button\" href=\"/docs\" target=\"_blank\">Docs<a/>';
+    template += '<a class=\"che-footer-button-blue che-footer-button\" href=\"/docs/factory/getting-started/index.html\" target=\"_blank\">Docs<a/>';
```

Comment:
It was 'Update links to docs in product'. That is why I changed general link in first time.